sycl/unittests/accessor/AccessorPlaceholder.cpp

```diff
@@ -1,6 +1,6 @@
 #include <gtest/gtest.h>
 
-#include <helpers/PiImage.hpp>
+#include <helpers/MockDeviceImage.hpp>
```

Comment:
Does the test need this include?

Reply:
Removed the include in 3aa939f4dd81b176fedb6303d41d7599b4acdced